web/src/components/merchant/sign-in/SignInCard.tsx

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import GroupBuyMerchantHeader from 'components/common/GroupBuyMerchantHeader';
+import SignInForm from 'components/merchant/sign-in/SignInForm';
+import Row from 'muicss/lib/react/row';
+import {Link} from 'react-router-dom';
+import styled from 'styled-components';
+
+const CardContainer: React.FC = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  background: var(--pale-gray);
+  border-radius: 30px;
+  max-width: 670px;
+  padding: 40px;
+  box-shadow: 4px 4px 10px rgba(0, 0, 0, 0.25);
+`;
+
+const StyledRow = styled(Row)`
+  height: 40px;
+  display: block;
+  line-height: 40px;
+`;
+
+/**
+ * This is the card containing the sign in form for the Merchant app.
+ * It is displayed in the Sign In page.
+ */
+const SignInCard: React.FC = () => {
+  return (
+    <CardContainer>
+      <GroupBuyMerchantHeader />
+      <SignInForm />
+      <StyledRow>
+        Already have an account? <Link to="/merchant/sign-up">Sign up</Link>{' '}
```

Comment:
The string should be not having an account?

Reply:
Oops, I forgot to change that! Thanks for pointing it out.

Reply:
+1. Can also add a reference back to sign in page from sign up page.

Reply:
There already is a reference from sign up page to sign in page. It's a similar link below the SIGN UP button.